.github/workflows/validateChangedFiles.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+INVALID_FILE_PATHS=('pkg/plugin/sdk') # Array of filepaths that are not allowed
+VALID_FILE_PATHS=('pkg/plugin/sdk/v1alpha2') # Array of filepaths that are allowed
+
+# CURRENT_BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
```suggestion
```

Reply:
Sorry for some of these things I had been planning to remove. Was just testing stuff. I had meant to make this a draft first